kuma/wiki/templatetags/ssr.py

```diff
@@ -0,0 +1,128 @@
+import json
+import subprocess
```

Comment:
The `subprocess` in stdlib Python 2.7 does not support timeouts.
Having done a lot of Python-to-Node subprocess stuff, it's only a matter of time until you get a dead process and Python ties up a blocked and neverdying process waiting for it.

Again, I don't know if the use of subprocess is a temporary hack but if we're going to land this in master we might as well switch to [`subprocess32`](https://pypi.org/project/subprocess32/) and use `import subprocess32 as subprocess`.

Reply:
I didn't know about subprocess32. If that enables me to add a timeouts for the reads and writes, that would be great.

I don't know if this is a temporary hack. After Ryan has had a chance to review, maybe we can all get together again to discuss how to move forward with this and how to integrate it with Ryan's JSON publishing work.

Reply:
It's a deal! I'll retract everything I said about `ssr.py` because I don't any of my concerns would remain valid. Having said that I'd rather not delete the comments. We'll just squint our eyes and ignore them being there because they shouldn't block a merge.